src/MongoDB/ReadPreference.c

```diff
@@ -369,6 +375,31 @@ static PHP_METHOD(ReadPreference, getTagSets)
 	}
 } /* }}} */
 
+static const char* php_phongo_readpreference_get_mode_string(mongoc_read_mode_t mode TSRMLS_DC) /* {{{ */
```

Comment:
I don't think this function needs to take `TSRMLS_DC`, as it doesn't call any internal PHP APIs that would require a `TSRMLS_CC`.